substrate/frame/revive/src/tests/pvm.rs

```diff
@@ -4017,7 +4020,7 @@ fn call_tracing_works() {
 	let (code, _code_hash) = compile_module("tracing").unwrap();
 	let (binary_callee, _) = compile_module("tracing_callee").unwrap();
 	ExtBuilder::default().existential_deposit(200).build().execute_with(|| {
-		let _ = <Test as Config>::Currency::set_balance(&ALICE, 100_000_000);
+		let _ = <Test as Config>::Currency::set_balance(&ALICE, 100_000_000_000_000);
```

Comment:
Additional weight should not lead to more balance required. There is no transaction fee collected here.